app/drivers/display/Kconfig

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) 2020 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+source "Kconfig.il0323"
```

Comment:
This should be `rsource` to source a file relative to the current file.

Reply:
ahhh... I was wondering what was going on there. I was dumbfounded as to what was going on and why I had to put the full local path to the file lol. Will fix this now.

Reply:
this has been changed